mock-centos.cfg

```diff
@@ -1,16 +1,18 @@
 # global options
-config_opts['cache_topdir'] = os.environ['CACHEDIR']
-config_opts['basedir'] = os.environ['CHROOT_DIR'] + '/mock'
-config_opts['use_bootstrap_container'] = False
+config_opts['chroot_setup_cmd'] = 'install tar gcc-c++ redhat-rpm-config redhat-release which xz sed make bzip2 gzip gcc coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep'
```

Comment:
Urgh, that's really how upstream does it :/

Reply:
Yes it's really bad!